ssh/tailssh/login1.go

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build linux || (darwin && !ios)
```

Comment:
this should be linux-only. None of this dbus stuff will actually work on darwin, even if it compiles.